cmd/monaco/integrationtest/v2/test-resources/integration-all-configs/project/openpipeline/events.json

```diff
@@ -0,0 +1,60 @@
+{
+  "customBasePath": "/platform/ingest/custom/events",
+  "editable": true,
+  "endpoints": [
+    {
+      "basePath": "/platform/ingest/v1/events",
+      "builtin": true,
+      "defaultBucket": null,
+      "displayName": "Default API",
+      "editable": false,
+      "enabled": true,
+      "processors": [],
+      "routing": {
+        "type": "dynamic"
+      },
+      "segment": ""
+    }
+  ],
+  "id": "events",
+  "pipelines": [
```

Comment:
Is this just the default configuration of this pipeline? If yes, could you please make this a custom configuration - e.g. by adding a pipeline, or such - so we can actually test applying a configuration and can't run into a situation where an optimized API just does nothing.

Reply:
I've added a second pipeline